### PR TITLE
Added information on how to use form flows to start a new case or sup…

### PR DESCRIPTION
**Pull request by ivo-ritense**
…porting process

### CODE REVIEW

File: reference/modules/form-flow.md [hunk 1]
```diff
@@ -129,4 +129,76 @@ The Camunda process will get an extra variable:
 }
 ```
 
+#### Starting a new case and saving submission data to defined location
 
+The method below creates a new case and saves the submission data to the document and process variables. Currently
```
Comment (theo-ritense) [inline reference/modules/form-flow.md:134]:
```suggestion
The method below creates a new case and saves the submission data to the document and process variables.
```

File: reference/modules/form-flow.md [hunk 2]
```diff
@@ -129,4 +129,76 @@ The Camunda process will get an extra variable:
...
+storing data in other location is not supported. The second parameter decides where in the document the submission data
```
Comment (theo-ritense) [inline reference/modules/form-flow.md:135]:
```suggestion
Storing data in other location is currently not supported. The second parameter decides where in the document the submission data 
```

File: reference/modules/form-flow.md [hunk 3]
```diff
@@ -129,4 +129,76 @@ The Camunda process will get an extra variable:
...
+```json
+{
+  "approved": true
+}
+```
+
+#### Starting a supporting process for a case
+
+The method below start a new process instance for an existing case and saves the submission data to the document and 
+process variables. Currently storing data in other location is not supported. The second parameter decides where in the
```
Comment (theo-ritense) [inline reference/modules/form-flow.md:171]:
```suggestion
process variables. Storing data in other location is currently not supported. The second parameter decides where in the 
```

Comment (ivo-ritense) [reply reference/modules/form-flow.md]: Changed

File: reference/modules/form-flow.md (hunk 2 above)
Comment (ivo-ritense) [reply reference/modules/form-flow.md]: Changed

File: reference/modules/form-flow.md (hunk 1 above)
Comment (ivo-ritense) [reply reference/modules/form-flow.md]: Changed

**Review by Klaas-Ritense — APPROVED** [review]

**Pull request merged**